src/body.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+
+import placeholder from './editor-placeholder';
+import TexttareaContainer from './editor';
+import ContainerPreview from './preview';
+
+const style = {
+  paddingTop: 20,
+};
+
+class ContainerPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      textarea: placeholder,
+      editorElementmaximized: false,
+      previewElementMaximized: false,
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleEditorClick = this.handleEditorClick.bind(this);
+    this.handlePreviewClick = this.handlePreviewClick.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({
+      textarea: event.target.value,
+    });
+  }
+
+  handleEditorClick() {
+    this.setState({
+      editorElementmaximized: !this.state.editorElementmaximized,
+    });
+  }
+
+  handlePreviewClick() {
+    this.setState({
+      previewElementMaximized: !this.state.previewElementMaximized,
+    });
+  }
+
+  render() {
+    const texttareaContainer = (
```

Comment:
remember we use camelcase convention, so this variable should be named: textAreaContainer